controllers/app.controller.js

```diff
@@ -1,14 +1,19 @@
 const {getAllTopics} = require("../models/app.model")
+const allEndpoints = require("../endpoints.json")
 
 
 function topicsController(request, response, next) {
     return getAllTopics().then((data) => {
-        return response.status(200).send(data);
+        return response.status(200).send({topics : data});
     })
     .catch(err => {
         next(err);
     })
 }
 
-module.exports = {topicsController}
+function apiController(request, response, next) {
+    response.status(200).send(allEndpoints);
+}
+
```

Comment:
I'd have a think about the name of your controller been something more like `getAllApiEndpoints` instead of `apiController`.

Similiar feedback for your topics controller (I know you're past that ticket), but you want your controller function name to be as close to the endpoint method and name as possible, for example your endpoint app.get("/api/topics"), the function name might be better as something along the lines of: `getTopics` or `getAllTopics`